index.html

```diff
@@ -3,34 +3,40 @@
   <head>
     <meta charset="utf-8">
     <title>Startrly</title>
+
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="styles.css">
   </head>
-  <body>
-    <div>
 
 
+  <body class="container">
+
+    <div>
+      <section class="first_block">
```

Comment:
I wonder if it would be more appropriate to have the block classes as id's since they only appear once in the file.